app/views/admin/legal_aid_applications/submissions/show.html.erb

```diff
@@ -19,12 +19,7 @@
     <% end %>
     <% if @legal_aid_application&.hmrc_responses.any? %>
       <% @legal_aid_application.hmrc_responses.each do |hmrc_response| %>
-        <details>
-          <summary class="govuk-heading-m govuk-details__summary"
-                   title="HMRC use case <%= hmrc_response.use_case %> <%= hmrc_response.owner_type %>"
-                   aria-label="HMRC use case <%= hmrc_response.use_case %> <%= hmrc_response.owner_type %> result">
-            HMRC response - use case <%= hmrc_response.use_case %> for the <%= hmrc_response.owner_type.downcase %>
-          </summary>
+        <%= govuk_details(summary_text: "HMRC response - use case #{hmrc_response.use_case} for the #{hmrc_response.owner_type.downcase}") do %>
```

Comment:
1. May be worth checking the summary text here is suitable from an accessibility point of view - I am looking at the hyphen and the differences in the deleted title, aria-label and content.
2. Be good to move to translations too if possible. happy to pair

```
<%= govuk_details(summary_text: t('.summary_text', use_case: hmrc_response.use_case, owner_type: hmrc_response.owner_type.downcase )) do %>
```

And...
```
# translation file
...
  summary_text: "HMRC response - use case %{use_case} for the %{owner_type}"
```

Reply:
but since this page is just for team members its not essential 🤷